src/parsers/manifest/dash/node_parsers/Period.ts

```diff
@@ -50,14 +52,79 @@ export interface IPeriodAttributes {
   xlinkActuate? : string;
 }
 
+/**
+ * Parse the EventStream node to extract Event nodes and their
+ * content.
+ * @param {Element} element
+ */
+function parseEventStream(element: Element): IStreamEvent[] {
```

Comment:
Why not in its own file, with its own little unit tests :D

Reply:
Now in it's own file